src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  border: #f3f3f3 1px solid;
+  border-radius: 5px;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.text {
+  height: 36px;
+  color: #060b35;
+  margin: 0 16px;
+  line-height: 18px;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 4px 0 0 16px;
+  line-height: 14px;
```

Comment:
don't forget to follow BEM in this task

Reply:
I did all these tasks, I just couldn't do the run deploy, so I uploaded them to a new request pull: https://github.com/mate-academy/layout_product-cards/pull/4959